docs/feature-store/end-to-end-demo/03-deploy-serving-model.ipynb

```diff
@@ -407,8 +407,8 @@
     "import mlrun.feature_store as fstore\n",
     "\n",
     "# Create the online feature service\n",
-    "svc = fstore.get_online_feature_service(\n",
-    "    \"transactions-fraud-short:latest\", impute_policy={\"*\": \"$mean\"}\n",
+    "svc = \"transactions-fraud-short:latest\".get_online_feature_service(\n",
```

Comment:
This line is wrong and will not compile by python

Reply:
What should it be?

Reply:
"svc = fstore.get_feature_vector(\"transactions-fraud-short:latest\").get_online_feature_service(impute_policy=...")